expat/CMakeLists.txt

```diff
@@ -611,6 +611,60 @@ else()
 endif()
 string(TOUPPER "${CMAKE_BUILD_TYPE}" _EXPAT_BUILD_TYPE_UPPER)
 
+#
+# packaging   
+#
+include(InstallRequiredSystemLibraries)
+
+set(CPACK_GENERATOR TGZ ZIP)
+if(WIN32 AND NOT CYGWIN)
+  set(CPACK_GENERATOR NSIS ${CPACK_GENERATOR})
+endif(WIN32 AND NOT CYGWIN)
```

Comment:
We already have a Windows installer based on Inno Setup, see:

- https://github.com/libexpat/libexpat/blob/master/expat/win32/expat.iss
- https://github.com/libexpat/libexpat/blob/master/expat/win32/build_expat_iss.bat
- https://github.com/libexpat/libexpat/commits/build-windows-installer-and-code-installed-by-installer

I've always liked NSIS (since the good old days of Winamp back when I was still using and developing for Windows) but we probably don't need two installers at the same time. It's questionable if we need an installer for DLL files in 2020 at all, it's been kept around to not take familiar things away. If we want to keep having an installer, we could either switch to NSIS and make the AppVeyor CI cover NSIS or stick with Inno Setup (and add xmlwf binaries or not). I'm happy to separate the installer topic from this pull request if you would prefer.

Reply:
NSIS was the decision for my project, lets follow your suggestion and separate the installer topic from this pull request.